src/js/common/schemaform/review/widgets.jsx

```diff
@@ -21,3 +22,18 @@ export function SelectWidget({ options, value }) {
 }
 
 export const RadioWidget = SelectWidget;
+
+export const yesNo = ({ value }) => {
```

Comment:
These fields are passed to the library as an object, so I was trying to avoid having to export an object and be able to do `import * as fields` in `FormPage`. That's probably not worth it, though.

Internally, the library maps these default widget names to a short name (e.g. `RadioWidget` becomes accessible as `'radio'` in the config). The non-review version of the yesNo widget follows their convention and sets `yesNo` as the key for `YesNoWidget` (in schemaform/widgets/index.js)